src/moonstone-samples/src/AlwaysViewingPanelsSample.js

```diff
@@ -13,7 +13,7 @@ module.exports = kind({
 	classes: 'moon enyo-fit enyo-unselectable',
 	style: 'background: gray url(\'http://lorempixel.com/1920/1080/\')',
 	components: [
-		{name: 'panels', kind: Panels, pattern: 'alwaysviewing', classes: 'enyo-fit', useHandle: false, components: [
+		{name: 'panels', kind: Panels, pattern: 'alwaysviewing', useHandle: false, hasCloseButton: false, components: [
```

Comment:
`ApplicationCloseButton` is not added in `alwaysviewing` pattern, so there's no effect in `hasCloseButton: false`